common/lib/xmodule/xmodule/tests/test_split_test_module.py

```diff
@@ -80,29 +82,35 @@ def setUp(self):
 
         self.module_system.descriptor_runtime = self.course._runtime  # pylint: disable=protected-access
         self.course.runtime.export_fs = MemoryFS()
+        self.split_test_module = self.course_sequence.get_children()[0]
 
+        # Create mock partition service
         user = Mock(username='ma', email='ma@edx.org', is_staff=False, is_active=True)
-        self.partitions_service = StaticPartitionService(
-            [
-                self.user_partition,
-                UserPartition(
-                    1, 'second_partition', 'Second Partition',
-                    [Group("0", 'abel'), Group("1", 'baker'), Group("2", 'charlie')],
-                    MockUserPartitionScheme()
-                )
-            ],
+        self.course.user_partitions = [
+            self.user_partition,
+            UserPartition(
+                1, 'second_partition', 'Second Partition',
+                [Group("0", 'abel'), Group("1", 'baker'), Group("2", 'charlie')],
+                MockUserPartitionScheme()
+            )
+        ]
+        partitions_service = MockPartitionService(
```

Comment:
@nasthagiri @cpennington Here is another example-- a PartitionService is being added onto module_system._services, and then module_system is bound to the split_test_module instance and user. module_system comes from get_test_system.